.github/actions/pytest_run/action.yml

```diff
@@ -5,10 +5,13 @@ runs:
   steps:
     - name: Test with pytest
       run: |
-        python -m pytest -rx -m "not parallel and c" --ignore=printers --ignore=symbolic --ignore=ndarrays
-        python -m pytest -rx -m "not parallel and not (c or python)" --ignore=printers --ignore=symbolic --ignore=ndarrays
-        python -m pytest ndarrays/ -rx
-        mpiexec -n 4 ${MPI_OPTS} python -m pytest epyccel/test_parallel_epyccel.py -v -m parallel -rx
-        #mpiexec -n 4 ${MPI_OPTS} python -m pytest epyccel -v -m parallel -rx
+        python -m pytest -n auto -rx -m "not (parallel or xdist_incompatible) and c" --ignore=printers --ignore=symbolic --ignore=ndarrays -x
+        python -m pytest -rx -m "xdist_incompatible and not parallel and c" --ignore=printers --ignore=symbolic --ignore=ndarrays
+        pyccel-clean
+        python -m pytest -n auto -rx -m "not (parallel or xdist_incompatible) and not (c or python)" --ignore=printers --ignore=symbolic --ignore=ndarrays
+        python -m pytest -rx -m "xdist_incompatible and not parallel and not (c or python)" --ignore=printers --ignore=symbolic --ignore=ndarrays
+        pyccel-clean
+        python -m pytest -n auto ndarrays/ -rx
+        pyccel-clean
```

Comment:
The `printers` directory was removed, correct?

The flag `--ignore=printers` should be removed from all of our commands.

Reply:
Fixed in cff9757